sdk/servicebus/Azure.Messaging.ServiceBus/src/Amqp/AmqpMessageExtensions.cs

```diff
@@ -19,6 +20,14 @@ public static AmqpMessage ToAmqpMessage(this ServiceBusMessage message)
             {
```

Comment:
Do we validate at the `ServiceBusMessage` level that there is only one body type?   If not, should we validate that here?

Reply:
No we don't do this, as this shouldn't be possible.

Reply:
Gotcha.   I see where I misread;  this was my mistake.